library/src/main/java/com/proxerme/library/connection/info/entity/Season.java

```diff
@@ -0,0 +1,145 @@
+package com.proxerme.library.connection.info.entity;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.IntRange;
+import android.support.annotation.NonNull;
+
+import com.afollestad.bridge.annotations.Body;
+import com.proxerme.library.parameters.SeasonParameter;
+import com.proxerme.library.parameters.TypeParameter;
+
+/**
+ * Class of the season entity of an entry.
+ *
+ * @author Desnoo
+ */
+public class Season implements Parcelable {
+
+    public static final Creator<Season> CREATOR = new Creator<Season>() {
+        @Override
+        public Season createFromParcel(Parcel in) {
+            return new Season(in);
+        }
+
+        @Override
+        public Season[] newArray(int size) {
+            return new Season[size];
+        }
+    };
+
+    @Body(name = "id")
+    String id;
+    @Body(name = "eid")
+    String entryId;
+    @Body(name = "type")
+    String type;
+    @Body(name = "year")
+    int year;
+    @Body(name = "season")
+    int season;
+
+    /**
+     * The private constructor.
+     */
+    @SuppressWarnings("unused")
```

Comment:
Are you sure we should start doing this? That would be maaaaany methods to annotate :D